docs/1manifoldLearning.md

```diff
@@ -0,0 +1,48 @@
+# 1-Manifold Learning example
+
+![1-Manifold Learning example Image](https://topology-tool-kit.github.io/img/gallery/1manifoldLearning.jpeg)
+
+
+## Pipeline description
+
+This example first loads a point cloud from disk. 
+
+In a pre-processing, the [DimensionReduction](https://topology-tool-kit.github.io/doc/html/classttk_1_1DimensionReduction.html) is used to reduce the dimension of the input to 3D points. The data is then converted to a format understandable by Paraview using the TableToPoints filter. Gaussian Resampling is applied to the data (upper left view in the above screenshot). This filter has the effect of injecting input points to a structured data. For each injection, each point will "splat", or distribute values to nearby vertices. The resulting SplatterValues field is density estimation (with a Gaussian kernel) of the point cloud projected in 2D.
```

Comment:
to 3D points -› to 2D.
(in fact, the input is just a 3D point cloud, and it's projected down to 2D)

Reply:
TableToPoints -› `TableToPoints`

Reply:
GaussianResampling -› `GaussianResampling`

Reply:
SplatterValues -› `SplatterValues`

Reply:
is density estimation -› is a density estimation